### DISCUSSION TITLE
Add support for additional inline-* display types in wire:loading

### DISCUSSION THREAD

**Discussion by smares** [livewire/livewire#3472] [3. Feature ideas]
The documentation lists the following display types that can be used with `wire:loading`:

```
<div wire:loading.flex>...</div>
<div wire:loading.grid>...</div>
<div wire:loading.inline>...</div>
<div wire:loading.table>...</div>
```

It would be nice if `inline-block`, `inline-flex` and `inline-grid` would be supported out of the box, too.

**Comment by joshhanley** [comment]
@smares you could actually do that using toggle classes option https://laravel-livewire.com/docs/2.x/loading-states#toggling-classes

For example if you are using tailwind:
```html
<div wire:loading.class="inline-flex">...</div>
```

Hope this helps!
